backend/routes/notes.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
```

Comment:
To be consistent with the rest of the repo, have some comments defining what these functions do.